old_go_warning.go

```diff
@@ -1,6 +1,5 @@
 // +build !go1.11
 
-"go-reuseport requires go >= 1.11"
-
-;
+package reuseport
```

Comment:
Added the package keyword because `gx-go` would complain and `make deps` would fail.

Reply:
:+1:

Reply:
So, I put that where it is so "go-reuseport requires go >= 1.11" would show up in the compiler's error message. However, I can confirm that this breaks gx so that's probably good enough.

Reply:
Probably. Note: there's nothing gx can do about this. Unfortunately, it has to ignore build tags (it can't know what build tags you're going to use later).